docs/features/configurationReload.md

```diff
@@ -3,6 +3,20 @@
 You have the following option to trigger a configuration reload:
 
 - via the user interface: `Manage Jenkins -> Configuration -> Reload existing configuration`
+- via http POST to `JENKINS_URL/reload-configuration-as-code/` (note the trailing slash).  
+    It's disabled by default and secured via a token configured either by the environment variable
+    `CASC_RELOAD_TOKEN` or the system property `casc.reload.token`. If both are set, the environment
+    variable takes precedence.  
+    Configuring this token enables this functionality and the requests need to include it as a
+    query parameter named `casc-reload-token`, e.g. `JENKINS_URL/reload-configuration-as-code/?casc-reload-token=someSecretValue`.
+
+```sh
+curl -X POST "JENKINS_URL/reload-configuration-as-code/?casc-reload-token=someSecretValue"
+
+# To avoid showing the token on the process list, one can use some bash/curl trickery:
```

Comment:
How about suggesting to set a local environment variable as a less trickery alternative?

(Thanks for it in any case, I didn't kew it)

Reply:
A local environment variable will still be expanded and show on the process list, as it's expanded by the shell:

```sh
angelo@irgeshud:~ > CASC_RELOAD_TOKEN=someSecretValue
# I used 10.0.0.0 to make sure curl hangs
angelo@irgeshud:~ > curl -X POST http://10.0.0.0:8080/reload-configuration-as-code/?casc-reload-token=$CASC_RELOAD_TOKEN 
# On another shell...
angelo@irgeshud:~ > pgrep -af curl
32701 curl -X POST http://10.0.0.0:8080/reload-configuration-as-code/?casc-reload-token=someSecretValue
```

With the trickery however:

```sh
angelo@irgeshud:~ > curl -X POST -G -d @<(cat <<< "casc-reload-token=$CASC_RELOAD_TOKEN") http://10.0.0.0:8080/reload-configuration-as-code/
# On another shell...
angelo@irgeshud:~ > pgrep -af curl
32940 curl -X POST -G -d @/dev/fd/63 http://10.0.0.0:8080/reload-configuration-as-code
```

I know it's unnecessary, but I had to do a LOT of googling to figure out that trickery. I could remove it and leave it to whomever is using to figure it out if you prefer.

Reply:
I just realized I can make the trickery extremely less trickery if one were to simply write the token to a file in the first place. In my head I couldn't write it to disk in the first place. I'll update it, should make it much simpler.